requests-for-comments/2024-08-14-youtuber.md

```diff
@@ -21,7 +21,30 @@ This does require the **right person**, a proper YouTuber, probably someone who
 
 **What does great look like?** YouTube grows as a source of signups. YouTuber is publishing multiple videos per week. Videos are consistently hitting 10k+ views, some viral hits. Developing video as expertise at PostHog.
 
-**Why not another (writing) content person?** There is always more newsletters, docs, tutorials, and blogs to do, but I don’t feel it is enough to add another content person (beyond filling in for James). 
+**Why not another (writing) content person?** We need to fill for James. There is always more newsletters, docs, tutorials, and blogs to do, but I don’t feel it is enough to add another content person (beyond filling in for James).
+
+## We shouldn’t wait for the community manager
+
+Review comments:
+
+> We should wait for the community manager
+
+I do see where you are coming from on a community manager front, this will likely overlap with what their work is, but I disagree on it being the reason we should not hire someone here reasons.
+
+1. The job listing for community manager went out June 7th-ish. This shows it takes a long time to hire for a role. It will take them more time to decide to join, onboard, and get used to the PostHog culture. It is a big unknown, both in terms of what they want to do and when it will happen. 
```

Comment:
Small nit here just so we're comparing correctly: the job ad was created then, but IIRC it wasn't public on the site for a fair old while as we were considering handpicked candidates only. The ad only actually went open to public applications in August - and we've had a steady stream of strong applications since.

Reply:
Fair! Didn't realize this.